src/CMakeLists.txt

```diff
@@ -271,9 +271,13 @@ if(WIN32)
 	find_path(ZLIB_INCLUDE_DIR "zlib.h" DOC "Zlib include directory")
 	find_library(ZLIB_LIBRARIES "zlib" DOC "Path to zlib library")
 
+	find_path(ZSTD_INCLUDE_DIR "zstd.h" DOC "Zstd include directory")
+	find_library(ZSTD_LIBRARY "zstd" DOC "Path to zstd library")
```

Comment:
why does Windows do this differently and not use the CMake module? :thinking:
not important though, can be changed later (along with Zlib, there's no reason for it do this)

Reply:
Yeah... I followed what zlib was doing.